addons/delivery/static/src/js/location_selector/location_selector_dialog/location_selector_dialog.scss

```diff
@@ -12,8 +12,14 @@
     }
 
     .o_location_selector_number {
-        width: 3ch;
+        right: .75rem;
+        bottom: .75rem;
+        width: 1.75rem;
         font-variant-numeric: tabular-nums;
```

Comment:
If we try to replicate the design of the pin in the map, I think we should also align the `font-size` and `font-family`. Right now, it looks similar but not totally the same. A bit weird IMO